README.md

```diff
@@ -26,6 +33,28 @@ The image below is an overwiev of the logic that our drone uses to process packe
 ![latest2](https://github.com/user-attachments/assets/68793d31-fc32-4103-8fcc-9bbc6711db44)
 > The colored squares represent how we split the drone logic in different functions and there are Diamond-shaped decision points for all of the relevant checks that our drone does when processing (except for the panics that are not included in the diagram)
 
+## usefull fonctions
+classical fonctions are implemented like **new** or **run**
+``` rust
+new(
+    id: NodeId,
+    controller_send: Sender<DroneEvent>,
+    controller_recv: Receiver<DroneCommand>,
+    packet_recv: Receiver<Packet>,
+    packet_send: HashMap<NodeId, Sender<Packet>>,
+    pdr: f32,
+    ) -> Self;
+
+run();
+```
+
+if you want you can also call functions like **set_pdr** or **add_channel***:
+```rust
+set_pdr(&mut self, pdr: f32)
+
+add_channel(&mut self, id: NodeId, sender: Sender<Packet>)
+```
+
```

Comment:
set_pdr and add_channel are private, they are not meant to be accessed outside of the drone, I think that this entire section is not needed because new and run are defined in the drone trait, which makes this a bit redundant, and we don't have any other public function